Mathlib/Algebra/Pointwise/Stabilizer.lean

```diff
@@ -132,6 +198,28 @@ end Finset
 
 /-! ### Stabilizer of a finite set -/
 
+variable {s : Set α}
+
+@[to_additive]
+lemma mem_stabilizer_iff_subset_smul_set (hs : s.Finite) : a ∈ stabilizer G s ↔ s ⊆ a • s := by
+  classical
+  lift s to Finset α using hs
+  norm_cast
+  exact mem_stabilizer_finset_iff_subset_smul_finset
+
+@[to_additive]
+lemma mem_stabilizer_iff_smul_set_subset (hs : s.Finite) : a ∈ stabilizer G s ↔ a • s ⊆ s := by
+  classical
+  lift s to Finset α using hs
+  norm_cast
+  exact mem_stabilizer_finset_iff_smul_finset_subset
+
+@[deprecated (since := "2024-11-25")]
+alias mem_stabilizer_of_finite_iff_smul_le := mem_stabilizer_iff_subset_smul_set
+
+@[deprecated (since := "2024-11-25")]
+alias mem_stabilizer_of_finite_iff_le_smul := mem_stabilizer_iff_smul_set_subset
+
 @[to_additive]
 lemma mem_stabilizer_set_iff_subset_smul_set {s : Set α} (hs : s.Finite) :
```

Comment:
It looks like this lemma is already in this file as `mem_stabilizer_set_iff_subset_smul_set`.

Reply:
That's *why* I had this feeling of déjà vu 😆